tests/baselines/reference/jsContainerMergeJsContainer.types

```diff
@@ -5,19 +5,19 @@ const a = {};
 >{} : {}
 
 a.d = function() {};
->a.d = function() {} : { (): void; prototype: {}; }
->a.d : { (): void; prototype: {}; }
+>a.d = function() {} : typeof d
+>a.d : typeof d
 >a : typeof a
->d : { (): void; prototype: {}; }
->function() {} : { (): void; prototype: {}; }
+>d : typeof d
+>function() {} : typeof d
 
 === tests/cases/conformance/salsa/b.js ===
 a.d.prototype = {};

```

Comment:
interesting...the change in type display indicates that we're now treating `a.d` as a class. I'm pretty sure that's because of the prototype assignment, where previously we only checked for members on the symbol.

(we print the type of 'a' as 'typeof a' because the expando assignments turn it into a namespace.)